server/index.js

```diff
@@ -1,9 +1,23 @@
+const path = require('path');
```

Comment:
`path` is a core part of Node, so you won’t find it in `package.json`. `path` contains several utilities for locating files in the filesystem.